src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import Footer from './Footer';
```

Comment:
Add blank line between these two import statements to indicate that the Footer is a local component/package